router/pkg/config/config.go

```diff
@@ -579,6 +580,11 @@ type PersistedOperationsConfig struct {
 	Storage PersistedOperationsStorageConfig `yaml:"storage"`
 }
 
+type ApolloCompatibilityFlags struct {
+	EnableAll       bool `yaml:"enable_all" envDefault:"false" env:"APOLLO_COMPATIBILITY_ENABLE_ALL"`
+	ValueCompletion bool `yaml:"value_completion" envDefault:"false" env:"APOLLO_COMPATIBILITY_VALUE_COMPLETION"`
```

Comment:
Do you think it makes sense to have an object here instead of a boolean? This would keep it open for extensions. In the future, we might have flags with additional configuration.